app/helpers/application_helper.rb

```diff
@@ -51,23 +51,17 @@ def multiple_with_filter?
   # it supports two formats :short and :long
   # example of long is February 12, 2021 17:13
   # example of short is Aug 31, 12:52
-  def date_time_absolute(time, format = :short)
```

Comment:
can you please extract all of the date/time methods to dedicated helper?

Reply:
I can do that but I'm not sure if some plugins aren't using it already. How about foreman-openscap. Is it used there @ares ?

Reply:
It's already used, which should not prevent the extraction. I don't understand why it should be in separate helper though? This is application wide helper method. If it was extracted to separate module, I'd still like to include it everywhere.